site/src/Map.jsx

```diff
@@ -222,27 +222,84 @@ export default function Map({
             type="symbol"
             source="divisions"
             source-layer="division"
-            filter={
-              ["all",
-                ["has", "@name"],
-                ["step",["zoom"],
-                  ["==","$type","Point"],
-                  2,["match",["get", "subtype"],["country","dependency"],true, false],
-                  4,["match",["get", "subtype"],["macroregion","region"],true, false],
-                  8,["match",["get", "subtype"],["macrocounty","county"],true, false],
-                  10,["match",["get", "subtype"],["county", "localadmin"],true, false],
-                  12,["match",["get", "subtype"],["localadmin", "locality", "borough", "macrohood", "neighborhood","microhood"],true, false]
-                ]
-              ]}
```

Comment:
Prettier formatting.

Reply:
(tangent but I'll review the CR to add prettier to avoid these diffs in the future)